contrib/storage-kafka/src/main/resources/bootstrap-storage-plugins.json

```diff
@@ -2,8 +2,8 @@
   "storage":{
     kafka : {
       type:"kafka",
-      enabled: false,
-      kafkaConsumerProps: {"bootstrap.servers":"localhost:9092", "group.id" : "drill-consumer"}
+      kafkaConsumerProps: {"bootstrap.servers":"localhost:9092", "group.id" : "drill-consumer"},
+      enabled: false
```

Comment:
is there any specific reason why this `enabled` setting is moved to end for all the plugins ?

Reply:
DRILL-6526 changed enabled config placement to be in the end of the json file for file storage plugins to be consistent with others, Vitalii just updated json files in project to be the same. You might be using not recent Drill version, anyway it doesn't really matter where enabled property is placed during deserialization.